lifecycle/services/delete.go

```diff
@@ -59,17 +58,17 @@ func (rbs *ReleaseBundlesService) RemoteDeleteReleaseBundle(rbDetails ReleaseBun
 	if err != nil {
 		return errorutils.CheckError(err)
 	}
+	hasDistributionRules := len(params.DistributionRules) > 0 && params.DistributionRules[0].GetSiteName() != "*"
 
-
-	restApi := GetRemoteDeleteReleaseBundleApi(rbDetails, isNewReleaseBundleApiSupported)
+	restApi := GetRemoteDeleteReleaseBundleApi(rbDetails, hasDistributionRules)
 	requestFullUrl, err := utils.BuildUrl(rbs.GetLifecycleDetails().GetUrl(), restApi, nil)
 	if err != nil {
 		return err
 	}
 
 	httpClientDetails := rbs.GetLifecycleDetails().CreateHttpClientDetails()
 	httpClientDetails.SetContentTypeApplicationJson()
-	if isNewReleaseBundleApiSupported {
+	if !hasDistributionRules {
 		resp, body, err := rbs.client.SendDelete(requestFullUrl, content, &httpClientDetails)
```

Comment:
is this the old API? what happens when we don't have distribution rules it uses `resp, body, err := rbs.client.SendPost(requestFullUrl, content, &httpClientDetails)`